src/main/res/layout-v21/view_server_file_item.xml

```diff
@@ -1,143 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
minSDKversion is 21. So why is this file removed?

Reply:
Since view_server_file_item.xml (v21) and view_server_file_item.xml (base) are the same, I removed the file with version qualifier so that naming is maintained. It does not matter which file is removed since both files are same.

Reply:
Great.